src/renderers/webgl/WebGLCapabilities.d.ts

```diff
@@ -11,7 +11,7 @@ export class WebGLCapabilities {
 		parameters: WebGLCapabilitiesParameters
 	);
 
-	readonly isWebGL2: true;
+	readonly isWebGL2: boolean;
```

Comment:
The lib usually set `true` for all `is*` properties. Hence, I would undo this particular change.